CONTRIBUTORS.md

```diff
@@ -0,0 +1,4 @@
+# Contributors
+
+* [Ben Kehoe](https://github.com/benkehoe)
+* [Geoff Baskwill](https://github.com/glb)
```

Comment:
You've raised the bar so much here that this is an extremely generous attribution 🙂

Reply:
Agree to disagree.